instance.tf

```diff
@@ -1,6 +1,6 @@
 resource "google_sql_database_instance" "instance" {
   database_version     = var.database_version
-  name                 = var.instance_name
+  name                 = "${var.project_prefix}-${var.instance_name}"
```

Comment:
If the default is null here then you will get `-${var.instance_name}` as the a result. You need to add `-` conditionally